src/components/dashboard/ColoredSection.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './ColoredSection.scss'
+
+class ColoredSection extends React.Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <div styleName="root">
+        { this.props.head == null ? null : (
```

Comment:
The first will not work when head is undefined.
The last suggestion would not work in general for falsy strings (in the case when the head is just a string).

I think in this place we should use ==